### ISSUE TITLE
Form does not work

### ISSUE THREAD

**Issue by uIIIastic** [MrTigreroux/TigerReports#81]
Hello! I ran into a problem when I create an empty blank in the DefaultReasons, the reasons behind it are not considered. They don't click stupidly. And only the menu is reloaded. What to do?

    Reason9: blank 
    Reason10:
      Name: "Chat rule violations"
      Lore: "&7The player is being disrespectful."
      # "&7The player is disrespectful."
      Item: 
      PunishCommands:
      - "mute _Reported_ 2h Insultes"

**Comment by MrTigreroux** [comment]
Hello,
You removed the Item type (after `Item:`), you must provide something like `Material-IRON_SWORD`.

**Comment by MrTigreroux** [comment]
If you can pick up the items in TigerReports menu that means another plugin disturbs TigerReports. Try to disable all your other plugins using menus (inventories) and test again TigerReports menu, then enable back one by one your plugins until you find the one disturbing TigerReports.

**Comment by uIIIastic** [comment]
I can select items from the menu, but I cannot select the items that come after: 
    Reason9: blank
all the buttons used after it are not pressed, but the menu itself is updated. The buttons that go before it work fine. When I disabled:
  CustomReasons: тщ
  Autoriser les raisons personnalisées
then she stopped accepting reports that come after the blank form. I turned on the ability to write any reasons for the reports, after which they started accepting. As I understand it, after a blank window in the reasons menu, it stops accepting any reasons that come after it. I think this has more to do with the plugin because it works great with custom reasons!

Sorry for my English. I don't know him well.
